tests/features/gpu_step_definitions_test.go

```
@@ -0,0 +1,1230 @@
+	return nil
+}
+
+// setupGPUTestResources creates Kueue resources for GPU testing
+func setupGPUTestResources(namespace string) error {
+	logDebug("Setting up GPU test resources in namespace: %s\n", namespace)
+
+	// Check if Kueue is installed
+	cmd := exec.Command("oc", "get", "crd", "clusterqueues.kueue.x-k8s.io")
+	if err := cmd.Run(); err != nil {
+		logDebug("Kueue not installed, skipping GPU resource setup\n")
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 179) but it returns nil

(nilerr)